sdk/src/gateway/client.ts

```diff
@@ -554,6 +563,55 @@ export class GatewayApiClient {
         return tokens.map((token) => ADDRESS_LOOKUP[this.chainId][token]).filter((token) => token !== undefined);
     }
 
+    // TODO: should get price from the gateway API
```

Comment:
can u create an issue in gateway repo with low prio for now.

Reply:
https://github.com/bob-collective/bob-gateway/issues/236